### ISSUE TITLE
Global scale size % for all slides

### ISSUE THREAD

**Issue by josephernest** [gnab/remark#620]
I have a document of around 50 slides, and I just noticed that with some browsers, the text does not fit in the window/browser viewport, and the bottom text of the slide is truncated.

* I have already tried to manually add things like `.remark-slide-content { font-size: 16px; }`, but it does not apply to h1, h2, h3, etc.

* I have already looked at https://github.com/gnab/remark/issues/104, but I haven't found an easy way to do this, without having to manually add many CSS rules.

**Is there a simple way to set a scale % for all the document?**

Example: I would like to set all slide at 85% (i.e. everything would be resized to 85% of the normal size, so that it would fit in the browser viewport).

Thanks in advance if you have an idea

**Comment by Katharsas** [comment]
try
```css
html {
font-size: 16px;
}
```

**Comment by josephernest** [comment]
@Katharsas Not really, it doesn't rescale everything in the document, as we could expect
